src/main/proto/netflix/titus/titus_vpc_api.proto

```diff
@@ -48,8 +48,41 @@ message ValidationRequest {
 message ValidationResponse {
 };
 
+message ValidationFailures {
+    repeated ValidationFailure failures = 1;
+}
+
+message ValidationFailure {
```

Comment:
IMHO, ths validation failures should offer structured and unstructured messages. Like:

oneof {
  NotFoundError notFoundError = 1;
  UnknownError unknownError = 2;
}

etc.?

Reply:
This is a generic failure message used by multiple validation APIs. 
Depending on the type of validation we are doing, the structured set could be quite different from one another. 
Do you think for the network allocation we have a closed set of such errors? I can create a separate API if you think that would be better.

Reply:
I am mistaken about this in my previous comment. This message is only for VPC API. 
Do we have a list of errors w.r.t. subnet, sg, and accountId validation that can constitute good first list?

Reply:
I do not think that is a good idea. I think we should have the response be validationfailures, and that contains repeated oneof many choice of validationfailure. If we take the oneof / enum approach, it cannot be reused, as the failure modes would be specific to this call.